src/Sentry/Laravel/EventHandler.php

```diff
@@ -149,6 +149,29 @@ protected function routeMatchedHandler(RouteMatched $match)
         $this->routerMatchedHandler($match->route);
     }
 
+    /**
+     * Until Laravel 5.1
+     *
+     * @param string $query
+     * @param array  $bindings
+     * @param int    $time
+     * @param string $connectionName
+     */
+    protected function queryHandler($query, $bindings, $time, $connectionName)
+    {
+        $data = array('connectionName' => $connectionName);
+
+        if ($this->sqlBindings) {
+            $data['bindings'] = $bindings;
+        }
+
+        $this->record(array(
```

Comment:
There is no `$this->record`
You need to add breacrumbs like `Integration::addBreadcrumb(`